### ISSUE TITLE
Bar Chart  transparent on the web

### ISSUE THREAD

**Issue by willianlouza** [indiespirit/react-native-chart-kit#733]
The Bar Chart is transparent on the web unless i reload the page.

The home page shows the Bar Chart normally

![Captura de tela de 2024-08-29 15-06-34](https://github.com/user-attachments/assets/09cd3ec6-78cf-45c3-8bdb-6e9009c1bd6f)

So if I navigate to another page which also has a Bar Chart, it displays transparent

![Captura de tela de 2024-08-29 15-06-51](https://github.com/user-attachments/assets/07471975-1c2c-45af-afa6-de7315ac818c)

And when I refresh the page, the graph is displayed as it should

![Captura de tela de 2024-08-29 15-06-56](https://github.com/user-attachments/assets/b146b4d9-ffd5-49bc-b647-50664d60106a)

 "expo-router": "~3.5.18"
 "react-native-chart-kit": "^6.12.0"
"react-native-svg": "^15.4.0"
"react-native-web": "~0.19.10"
"expo": "~51.0.22"
"react-native": "0.74.3"

**Comment by willianlouza** [comment]
`
const chartConfig: Partial<AbstractChartConfig> = {
  backgroundColor: CMEXX_COLORS.TERTIARY,
  backgroundGradientFrom: CMEXX_COLORS.TERTIARY,
  backgroundGradientTo: CMEXX_COLORS.TERTIARY,
  backgroundGradientFromOpacity: 0.2,
  backgroundGradientToOpacity: 0.1,
  color: (opacity = 1) => CMEXX_COLORS.PRIMARY,
};
...

 return (
    <View
      style={{
        marginVertical: 16,
        justifyContent: "center",
        alignItems: "center",
      }}
    >
      <Text style={{ marginBottom: 8 }}>{classificacao.nome}</Text>
      <Text>{mounted}</Text>
      <View
        style={{
          borderWidth: 1,
          borderRadius: 6,
          borderColor: CMEXX_COLORS.PRIMARY,
        }}
      >
        <BarChart
          style={{ paddingRight: 0 }}
          data={barChartData}
          width={(Platform.OS === "web" ? screenWidth / 2 : screenWidth) - 40}
          withHorizontalLabels={false}
          height={220}
          chartConfig={{
            ...chartConfig,
            propsForLabels: {
              fontSize: RFPercentage(1.2),
              fontFamily: "SpaceMono",
            },
          }}
          withInnerLines={false}
          withCustomBarColorFromData={false}
          yAxisLabel=""
          yAxisSuffix=""
          showValuesOnTopOfBars
        />
      </View>
    </View>
  );


`

**Comment by willianlouza** [comment]
```
const chartConfig: Partial<AbstractChartConfig> = {
  backgroundColor: CMEXX_COLORS.TERTIARY,
  backgroundGradientFrom: CMEXX_COLORS.TERTIARY,
  backgroundGradientTo: CMEXX_COLORS.TERTIARY,
  backgroundGradientFromOpacity: 0.2,
  backgroundGradientToOpacity: 0.1,
  color: (opacity = 1) => CMEXX_COLORS.PRIMARY,
};

....

 return (
    <View
      style={{
        marginVertical: 16,
        justifyContent: "center",
        alignItems: "center",
      }}
    >
      <Text style={{ marginBottom: 8 }}>{classificacao.nome}</Text>
      <View
        style={{
          borderWidth: 1,
          borderRadius: 6,
          borderColor: CMEXX_COLORS.PRIMARY,
        }}
      >
        <BarChart
          style={{ paddingRight: 0 }}
          data={barChartData}
          width={(Platform.OS === "web" ? screenWidth / 2 : screenWidth) - 40}
          withHorizontalLabels={false}
          height={220}
          chartConfig={{
            ...chartConfig,
            propsForLabels: {
              fontSize: RFPercentage(1.2),
              fontFamily: "SpaceMono",
            },
          }}
          withInnerLines={false}
          withCustomBarColorFromData={false}
          yAxisLabel=""
          yAxisSuffix=""
          showValuesOnTopOfBars
        />
      </View>
    </View>
  );
```

**Comment by hummus-erectus** [comment]
I am experiencing this too. I don't suppose you found a solution?

**Comment by willianlouza** [comment]
No. I'm using [react-native-gifted-charts](https://www.npmjs.com/package/react-native-gifted-charts) instead

**Comment by jyotykumari63** [comment]
@willianlouza [react-native-gifted-charts](https://www.npmjs.com/package/react-native-gifted-charts) pie chart is not working with onPress/focusOnPress event on web? any suggestion

**Comment by Jelmerovereem** [comment]
Same here, any fix?

**Comment by Jelmerovereem** [comment]
I was able to fix it in my fork f81c257df204b7979459ca5a37c6ffb1ce8a0d29.

Here is a patch file for [patch-package](https://github.com/ds300/patch-package): 

[react-native-chart-kit+6.12.0.patch](https://github.com/user-attachments/files/22367017/react-native-chart-kit%2B6.12.0.patch)
